kolibri/content/serializers.py

```diff
@@ -25,6 +25,7 @@ class ChannelMetadataSerializer(serializers.ModelSerializer):
     available = serializers.SerializerMethodField()
 
     def to_representation(self, instance):
+        # TODO: rtibbles - cleanup this for device specific serializer.
```

Comment:
All the device content references should be using the device specific one now, so this functionality should be able to be moved onto a separate serializer for that (rather than sharing a serializer).